NativeAuthSampleApp/MultiFactorAuthenticationViewController.swift

```diff
@@ -155,7 +161,43 @@ extension MultiFactorAuthenticationViewController: SignInStartDelegate {
         }))
 
         alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: { _ in
-            self.resultTextView.text = "Second factor authentication required"
+            self.resultTextView.text = "Second factor authentication is required"
+        }))
+
+        present(alert, animated: true)
+    }
+    
+    func onSignInStrongAuthMethodRegistration(authMethods: [MSALAuthMethod], newState: RegisterStrongAuthState){
+        print("SignInStartDelegate: onSignInStrongAuthMethodRegistration")
+        
+        showResultText("Just In Time Registration is required")
```

Comment:
Can you update this text? 
```suggestion
        showResultText("Stong authentication method registration is required")
```